VAMobile/src/utils/secureMessaging.tsx

```diff
@@ -480,7 +480,12 @@ export const saveDraftWithAttachmentAlert = (
 
 export const getLinkifiedText = (body: string, t: TFunction): ReactNode => {
   const textReconstructedBody: Array<ReactNode> = []
-  const bodySplit = body.split(/\s/)
+  const bodySplit = body.split(/\s/).filter((value) => value !== '')
+  const whiteSpace = body
+    .trim()
+    .split(/[^\s]/)
```

Comment:
I think this could be
```
.split(/\S/)
```
instead, a bit simpler

Reply:
updated